composer.json

```diff
@@ -25,9 +25,14 @@
     "require": {
         "php": ">=7.3",
         "composer/installers": ">=1.0.12",
-        "paytrail/paytrail-php-sdk": ">=1",
+        "paytrail/paytrail-php-sdk": ">=2",
         "ext-mbstring": "*",
         "symfony/polyfill-intl-idn": "v1.20.0",
```

Comment:
symphony packages, and `composer/installers` can be removed entirely from composer.
Symphony packages could conflict with other packages, and will be installed if needed with sdk.